### ISSUE TITLE
Discord Canary Setup does not work.

### ISSUE THREAD

**Issue by BurnedPinguin** [vxiiduu/VxKex#139]
As claimed in the latest release notes, Discord Canary should work. However, setup doesn't work:
![image](https://github.com/vxiiduu/VxKex/assets/77156464/9c00bdf3-8ef8-4415-9138-4e2fefd9a4c6)

The same issue is exhibited with regular Discord Setup and with Update.exe from an already installed Discord instance.

**Comment by cutiegin** [comment]
u can use Vesktop its a discord fork and its works with VxKex

**Comment by vxiiduu** [comment]
The installer worked for me but if you can't get it to work, you can extract the installer using 7-Zip.

![image](https://github.com/vxiiduu/VxKex/assets/73044267/f4a1b7a2-208e-4458-8915-bd325894c6ce)

I plan to get chrome, firefox, brave, edge etc. installers working at some point as well.

**Comment by BurnedPinguin** [comment]
> The installer worked for me but if you can't get it to work, you can extract the installer using 7-Zip.
> 
> ![image](https://private-user-images.githubusercontent.com/73044267/314237816-f4a1b7a2-208e-4458-8915-bd325894c6ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NDcwMjUsIm5iZiI6MTcxMDk0NjcyNSwicGF0aCI6Ii83MzA0NDI2Ny8zMTQyMzc4MTYtZjRhMWI3YTItMjA4ZS00NDU4LTg5MTUtYmQzMjU4OTRjNmNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIwVDE0NTg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZjJmZTlkMjJmYTc0YzIyYjFhNTg0NjhlMDgyYzY4YWIyMWQ3YjE2NjljOWMzMDg2NTI0OGM3YTUwZDI1MjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LdACWas8Wh_AL6RAbdRxOITeIhYFZIIdZA0mdI-ZKdo)
> 
> I plan to get chrome, firefox, brave, edge etc. installers working at some point as well.

No, that does not work. I get the same error, 0xc0000142. I did some testing, this error replaces the msvw10.dll error. Every program that exhibited the msvw10.dll issue now gives this error instead.

![image](https://github.com/vxiiduu/VxKex/assets/77156464/dee0ca76-dc8c-4eee-aeef-7458391417ef)

**Comment by RogalDoe** [comment]
> > The installer worked for me but if you can't get it to work, you can extract the installer using 7-Zip.
> > ![image](https://private-user-images.githubusercontent.com/73044267/314237816-f4a1b7a2-208e-4458-8915-bd325894c6ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA5NDcwMjUsIm5iZiI6MTcxMDk0NjcyNSwicGF0aCI6Ii83MzA0NDI2Ny8zMTQyMzc4MTYtZjRhMWI3YTItMjA4ZS00NDU4LTg5MTUtYmQzMjU4OTRjNmNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIwVDE0NTg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThlZjJmZTlkMjJmYTc0YzIyYjFhNTg0NjhlMDgyYzY4YWIyMWQ3YjE2NjljOWMzMDg2NTI0OGM3YTUwZDI1MjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LdACWas8Wh_AL6RAbdRxOITeIhYFZIIdZA0mdI-ZKdo)
> > I plan to get chrome, firefox, brave, edge etc. installers working at some point as well.
> 
> No, that does not work. I get the same error, 0xc0000142. I did some testing, this error replaces the msvw10.dll error. Every program that exhibited the msvw10.dll issue now gives this error instead.
> 
> ![image](https://private-user-images.githubusercontent.com/77156464/314550627-dee0ca76-dc8c-4eee-aeef-7458391417ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwNDUwNTcsIm5iZiI6MTcxMTA0NDc1NywicGF0aCI6Ii83NzE1NjQ2NC8zMTQ1NTA2MjctZGVlMGNhNzYtZGM4Yy00ZWVlLWFlZWYtNzQ1ODM5MTQxN2VmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIxVDE4MTIzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg0MTFiYjY5Mjc0NTY3ZGIyZjY0ZGE1ODAwYjlmYjQxYmEyMGNkM2ViMzYyOWJjYTJiN2MxZjAxMWY4ZmFlMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tqlcyTfzwmwjyfBnHdefwEPUZCDLepQa7Suyn6H4Qpo)

---------------------------
DiscordCanary.exe - Error System
---------------------------
The application was unable to start correctly missing msvw10.dll on your computer 
---------------------------
OK   
---------------------------

**Comment by RogalDoe** [comment]
yeah have the same error on my test computer then don't know where are this dependency searched in .exe and .dll but this dependency sems to be on pak file or somewhere like this

**Comment by BurnedPinguin** [comment]
Here are the logs I get with 1.1.1.1375 using discord regular's update.exe, discord canary's setup, and discord canary's discord.exe.
[Logs.zip](https://github.com/vxiiduu/VxKex/files/14722155/Logs.zip)

**Comment by BurnedPinguin** [comment]
Discord currently works under VxKex only if you use a 64-bit version of Discord. The best way to guarantee that is to use Canary, but if you manage to find any other 64-bit versions of Discord, they will work as well. But you must only enable VxKex extensions, and do not enable version spoof. From what I can see, you will still get serviced updates, even though Discord will tell you your Windows is not up to date.

**Comment by RogalDoe** [comment]
> Discord currently works under VxKex only if you use a 64-bit version of Discord. The best way to guarantee that is to use Canary, but if you manage to find any other 64-bit versions of Discord, they will work as well. But you must only enable VxKex extensions, and do not enable version spoof. From what I can see, you will still get serviced updates, even though Discord will tell you your Windows is not up to date.

I found a way to run Canary, PTB, Stable in a single app ( VENCORD )


And is including Better discord on it

**Comment by atylo** [comment]
Well, I found a way to make it work, along with Vencord. You download 64-bit Canary from here
https://support.discord.com/hc/en-us/articles/360035675191-Discord-Testing-Clients
Then you check VxKex in VxKex Properties, and after the install, you also check VxKex for the DiscordCanary.exe
In C:\Users\user\AppData\Local\DiscordCanary\app-x.x.x 
To install Vencord, you download VencordInstallerCli.exe from Vencord website and install it on Discord canary. I checked VxKex option for it too just in case.
